setup.py

```diff
@@ -69,6 +69,8 @@ def find_version(*pathnames):
         "scikit-image==0.13.1",
         "scikit-learn==0.19.0",
         "scipy==1.0.1"
```

Comment:
@erindiel commas are missing for all the deps here, hence the build is failing.

Reply:
Thanks :) Fixed and pip installs on my machine now.